interview-questions.md

```diff
@@ -10,48 +10,116 @@ Then, research the question to expand on your answer. Even if you feel you have
 
   Your answer:
 
+  A parameter is a named variable passed into a function. Arguments are objects used to run the functions.
+
   Researched answer:
 
+The arguments is an object which is local to a function. This object (arguments) is used to access the parameter passed to a function. It is only available within a function. This object (arguments) is used to access the parameter passed to a function. It is only available within a function. Arguments object allow you to access all of the arguments that are passed to a function. We can access these arguments using indexes.
```

Comment:
To add some clarity, arguments are data that is passed into a function when it is initially called or invoked.  A param becomes an alias or placeholder name for the value which was passed in.  Ex: 
`const myFunction = (param) => { return param }
console.log(myFunction(argument))`